.github/workflows/test.yml

```diff
@@ -0,0 +1,20 @@
+---
+on: [push, pull_request]
```

Comment:
let's make this only on pull_request, there's a workflow for publishing to npm on push lest add the build step to that instead

Reply:
Done.